taxonomies/ingredients.txt

```diff
@@ -1899,9 +1899,9 @@ en:E1104
 en:transglutaminase
 ar:ترانسغلوتاميناز
 bg:трансглутаминаза
-ca:transglutaminasa
+ca:transglutaminasa, agent de tractament de la farina
```

Comment:
There is:
en: Flour treatment agent
in 
additives_classes.txt

Reply:
It creates dupliclicates otherwise
```
2023-12-24T19:35:00.3759629Z ERROR - ca:agent-de-tractament-de-la-farina already is a synonym of ca:agent-de-tractament-de-la-farina for entry en:flour-treatment-agent (ingredients) - ca:agent-de-tractament-de-la-farina cannot be mapped to entry en:transglutaminase / ca:transglutaminasa
2023-12-24T19:35:00.3762096Z ERROR - es:agente-de-tratamiento-de-la-harina already is a synonym of es:agente-de-tratamiento-de-la-harina for entry en:flour-treatment-agent (ingredients) - es:agente-de-tratamiento-de-la-harina cannot be mapped to entry en:transglutaminase / es:transglutaminasa
```

Reply:
Current version in prod that has this `additive` defined, does not parse, this is why I added it.

![image](https://github.com/openfoodfacts/openfoodfacts-server/assets/1145001/371e9c9c-0697-470a-81de-d8e81df1ffcb)

Reply:
ah, okay. I didn't know.

I think it is because of the "i" ("and"). When there is "and" it parses only if both left and right ingredients are recognized. I tried to replace by comma (locally) and get the following result:
![Screenshot_20231225_194025](https://github.com/openfoodfacts/openfoodfacts-server/assets/110821832/0413070d-e89a-4668-b2dd-f7f5ec10d602)